express-notes-api/index.js

```diff
@@ -0,0 +1,91 @@
+const express = require('express');
+const app = express();
+const dataJSON = require('./data.json');
+const fs = require('fs');
+
+app.get('/api/notes', (req, res) => {
+  const arr = [];
+  for (const key in dataJSON.notes) {
+    arr.push(dataJSON.notes[key]);
+  }
+  res.json(arr);
+  res.sendStatus(200);
```

Comment:
you can leave out `sendStatus` here because

a) the default status is 200 if unspecified

b) the `.json()` method already sent the response so the train has already left the station for setting any specific status code